spark-cobol/src/main/scala/za/co/absa/cobrix/spark/cobol/source/parameters/CobolParametersParser.scala

```diff
@@ -45,6 +45,7 @@ object CobolParametersParser {
   val PARAM_GENERATE_RECORD_ID        = "generate_record_id"
   val PARAM_SCHEMA_RETENTION_POLICY   = "schema_retention_policy"
   val PARAM_GROUP_FILLERS             = "drop_group_fillers"
+  val PARAM_GROUP_NOT_TERMINALS       = "non_terminals"
```

Comment:
Would you mind adding the parameter to README.md as well?

It is optional, we can do it later, before the release.